Server/VikingVault.Services/PaymentTogglingService.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using VikingVault.DataAccess;
+using VikingVault.Services.Abstractions;
+
+namespace VikingVault.Services
+{
+    public class PaymentTogglingService : IPaymentTogglingService
+    {
+        private VikingVaultDbContext _context;
+
+        public PaymentTogglingService(VikingVaultDbContext context)
+        {
+            _context = context;
+        }
+
+        public void ChangePaymentState(int id, string value)
+        {
+            var automaticPayment = _context.AutomaticPayments.Find(id);
+            automaticPayment.IsEnabled = !bool.Parse(value);
```

Comment:
Why not receive the boolean value directly instead of receiving it as a string and then parsing it?